docs/src/main/sphinx/installation/deployment.md

```diff
@@ -32,10 +32,10 @@
 (requirements-java)=
 ### Java runtime environment
 
-Trino requires a 64-bit version of Java 22, with a minimum required version of
-22.0.1 and a recommendation to use the latest patch version. Earlier major versions
-such as Java 8, Java 11, Java 17 or Java 21 do not work. Newer versions such as
-Java 23 are not supported -- they may work, but are not tested.
+Trino requires a 64-bit version of Java 23, with a minimum required version of
+23.0.0 and a recommendation to use the latest patch version. Earlier versions
```

Comment:
not 23.0.1 ?

Reply:
No, we updated to 23.0.1 but require 23.0.0 as a min version